split_log

```diff
@@ -0,0 +1,23 @@
+#!/bin/python3
+
+# usage: split_log [output_stats_file] [output_info_file]
+
+import sys
+import os
+import pathlib
+
+stats_file = sys.argv[1]
+info_file = sys.argv[2]
+
+pathlib.Path(os.path.dirname(stats_file)).mkdir(parents=True, exist_ok=True)
+pathlib.Path(os.path.dirname(info_file)).mkdir(parents=True, exist_ok=True)
+
+line = bytearray()
+while True:
+  b_in = sys.stdin.buffer.read1(1)
+  line.extend(b_in)
+  if b_in == b'\n' or b_in == b'\r':
+    out_file = stats_file if line.decode().startswith('frame') else info_file
+    with open(out_file, 'ab+') as f:
+      f.write(line)
+    line = bytearray()
```

Comment:
Can we move this into ASM_Common?  See corresponding comments in ASM-data-server